spec/index.bs

```diff
@@ -7960,6 +7960,25 @@ dictionary GPURenderPassColorAttachment {
         render pass. If not [=map/exist|provided=] defaults to `{r: 0, g: 0, b: 0, a: 0}`. Ignored
         if {{GPURenderPassColorAttachment/loadOp}} is not {{GPULoadOp/"clear"}}.
 
+        The members of {{GPURenderPassColorAttachment/clearValue}} are all double values, so
+        they will first be converted to the fully qualified format type of
+        {{GPURenderPassColorAttachment/view}} before being set as the clear value of
+        {{GPURenderPassColorAttachment/view}}.
+
+        Let |colorAttachmentFormat| be
+        {{GPURenderPassColorAttachment/view}}.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}}.
+        - If the components of |colorAttachmentFormat| are floating point values, the members of
```

Comment:
> If the components

nit: Since each component can be a slightly different type, e.g. rgb10a2unorm, I would tweak the language here slightly to define in terms of a single component (scalar input value and component type) and say it applies to each component separately.

Reply:
Now I am using the concept of "channel" (defined in WGSL SPEC) here.